JobConfig/cosmic/prolog.fcl

```diff
@@ -177,11 +185,16 @@ Cosmic: {
       }
     ]
   }
-  MultiStageParametersCosmicS2: {
+  S2Inputs: {
     primaryType: StepPoints
```

Comment:
@dnbrow01 , didn't you want to save CrvSteps on S1 as well?

Reply:
They aren't created in S1.  We could add that.  That's probably the best thing to do, then the Low filter can run off of those instead of the StepPointMCs.